CHANGELOG.md

```diff
@@ -1,5 +1,61 @@
+  * Network-isolated clusters are now Generally Available, including GA support for CNIv1 → Cilium and Cilium → Overlay upgrade paths.
+
+* Bug Fixes  
+  * Retained `kube-proxy` on add-ons v2 during Cilium migrations to prevent transient service-connectivity loss.  
+  * Updated the Windows GPU device-plugin to `v0.0.19`, mitigating CVE-2025-22871.  
+  * Switched the static-egress-gateway init container to the pre-built image `v0.0.20`, eliminating failures in environments without Internet access.  
+  * Re-enabled the `ExtensionManager` on Overlay, unblocking Flux extension deployments.  
+  * Overlay manager now ignores maintenance windows that aren’t weekly.  
+  * Fixed failures triggered by duplicate tag keys that differed only by character case.
+
+* Behavior Changes  
+  * In-place upgrades are now supported when the cluster is using [Azure CNI Overlay (preview)](https://learn.microsoft.com/azure/aks/azure-cni-overlay).
+  * Overlay manager disregards non-weekly maintenance windows (see Bug Fixes).  
+  * Duplicate tag keys that differ only by case are now rejected at creation time.
+
+* Component Updates  
+  * Updated NAP Custom Resource Definitions to [`1.4.0-aks`](https://github.com/Azure/karpenter-provider-azure/blob/main/charts/karpenter-crd/templates/karpenter.azure.com_aksnodeclasses.yaml)
+    * Added zone-label support and image-status fields.  
+    * Introduced `v1beta1` API for AKSNodeClass; default CRs updated accordingly.  
+    * Deployment flow now consumes `vnetGUID` from Secret when present.  
+    * Clean-up of obsolete resource versions/labels; broader ConfigMap read permissions for maintenance-window detection.  
+  * Windows node images  
+    * Server 2019 Gen1 – [`17763.7240.250416`](vhd-notes\AKSWindows\2019\17763.7240.250416.txt)
+    * Server 2022 Gen1/Gen2 – [`20348.3561.250416`](vhd-notes\AKSWindows\2022\20348.3561.250416.txt)
+    * Server 23H2 Gen1/Gen2 – [`25398.1551.250416`](vhd-notes\AKSWindows\23H2\25398.1551.250416.txt)
+  * AgentBaker versions bumped to [`v0.20250422.0`](https://github.com/Azure/AgentBaker/releases/tag/v0.20250422.0), [`v0.20250422.0`](https://github.com/Azure/AgentBaker/releases/tag/v0.20250428.0), [`v0.20250509.0`](https://github.com/Azure/AgentBaker/releases/tag/v0.20250509.0), and [`v0.20250514.0`](https://github.com/Azure/AgentBaker/releases/tag/v0.20250514.0).  
+  * Container Insights Linux DaemonSet memory limit increased to 1 GiB.  
+  * kubelet-serving-csr-approver plug-in enabled by default in all public regions; new Helm chart released.  
+  * Cilium introduces a new (disabled-by-default) metric `ciliumproxydatapathupdatetimeout_total`.  
+  * KSCR (Kubelet Service Certificate Rotation) is now enabled by default in West Central US and East Asia.  
+  * Confidential VM support added for Ubuntu 24.04 running on Azure Linux 3.0.  
```

Comment:
- 'Ubuntu 24.04 running on Azure Linux 3.0' - what does this mean?
- docs link is missing. on which, our CVM docs is still listing Azure Linux under limitations which the above line is differing on.

Reply:
No its only for Ubuntu you are right the AZLinux is some AI add. i ll remove it.